docs/src/gallery/guides.md

```diff
@@ -67,6 +67,28 @@ plot(Dsleep, x=:BodyWt, y=:BrainWt, Geom.point, color=:Vore, shape=:SleepTime,
 ```
 
 
+## [`Guide.sizekey`](@ref)
+
+```@example
+using Compose, Gadfly, RDatasets
+set_default_plot_size(14cm, 8cm)
+
+Titanic = dataset("datasets", "Titanic")
+Class =  by(Titanic, :Class, :Freq=>sum)
+Titanic = join(Titanic[Titanic.Survived.=="Yes",:], Class, on=:Class)
+Titanic.prcnt = 100*Titanic.Freq./Titanic.Freq_sum
+sizemap = n->range(3pt, 8pt, length=n)
+
+plot(Titanic, Scale.x_log10,  Scale.y_log10,
+    x=:Freq, y=:prcnt, color=:Age, shape=:Sex,  size=:Class,
+    Scale.size_discrete2(sizemap), Guide.sizekey(title="Passenger\n Class"),
+    Guide.colorkey(pos=[0.1, -0.3h]), Guide.shapekey(pos=[0.5, -0.31h]),
+    Guide.ylabel("% of Passenger Class"),
+ Theme(discrete_highlight_color=identity, alphas=[0.1], key_swatch_color="grey",
+    key_swatch_shape=Shape.circle, point_size=3pt) )
```

Comment:
a near identical example appears in gallery/scales, albeit without the key.  is there another dataset that could be used here?  the more variety in the gallery the more likely the reader might find something closer to what they're looking for.

Reply:
With this PR, the size key appears* in the example in gallery/scales, but there are a few differences including:
- key title is different (Class changed to "Passenger\nClass" in gallery/guides)
- the swatch shape is different (squares is the default)
- `levels` is set in `Scale.size_discrete` in gallery/scales

Thus the differences between the two examples in gallery/scales and gallery/guides illustrates to users the difference between a Scale and a Guide.

*Note that the size key is triggered in the same way as the color keys, i.e the size key is **not** triggered by one element vectors e.g. `size=[1.41]`